h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts

```diff
@@ -253,14 +254,17 @@ export class SummaryComponent implements OnInit {
     }
 
     // remove password
-
     if (results['operations']['opsPassword']) {
-      results['operations']['opsPassword'] = '*';
+      delete results['operations']['opsPassword'];
     }
 
     return results;
   }
 
+  escapeSpecialCharsForCLI(text) {
+    return text.replace(/([() ])/g, '\\$&');
```

Comment:
For now only including what we know we need to escape which is () and whitespace in between words.